src/transformers/modeling_utils.py

```diff
@@ -1010,6 +1029,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         mirror = kwargs.pop("mirror", None)
         from_pipeline = kwargs.pop("_from_pipeline", None)
         from_auto_class = kwargs.pop("_from_auto", False)
+        _no_fast_init = kwargs.pop("_no_fast_init", False)
```

Comment:
To prevent breaking backwards compatibility we should probably set this to `True` for now until 5.0 is out.

Reply:
IMO it's fine if there is breaking change in random initialization in a minor release, especially as it will speed things up. Especially since you went out of your way to provide an argument to replicate the previous behavior. We can document it properly in the release notes.

Reply:
And since v5.0 is who knows when, this means that this feature won't be accessible to example scripts, unless we code a special flag to override such default. 

So if we make this True, there must be a transparent way to override it - e.g. via temp env var until 5.0 is released.

Reply:
Agreed for this change